test/unit/markdown-it/index.test.js

```diff
@@ -0,0 +1,61 @@
+const markdownIt = require('../../../src/lib/markbind/src/lib/markdown-it/');
+
+// test('renders 1', () => {
```

Comment:
Why are these commented out?

If you want, you can use `test.skip` instead to skip unwanted tests.

Reply:
This PR is still WIP. 😅 I planned to delete this.

Reply:
Oh oops didn't see the wip